legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-snowflake/legend-engine-xt-relationalStore-snowflake-pure/src/main/resources/core_relational_snowflake/relational/sqlQueryToString/snowflakeExtension.pure

```diff
@@ -158,9 +158,11 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::snowf
     dynaFnToSql('hour',                   $allStates,            ^ToSql(format='date_part(\'hour\', %s)')),
     dynaFnToSql('indexOf',                $allStates,            ^ToSql(format='CHARINDEX(%s)', transform={p:String[2] | $p->at(1) + ', ' + $p->at(0)})),
     dynaFnToSql('isAlphaNumeric',         $allStates,            ^ToSql(format=regexpPattern('%s'), transform={p:String[1]|$p->transformAlphaNumericParamsDefault()})),
+    dynaFnToSql('jaroWinklerSimilarity',  $allStates,            ^ToSql(format='JAROWINKLER_SIMILARITY(%s, %s)')),
```

Comment:
We need implementations for other DBs as well. Especially H2, DuckDB (for unit tests)

Reply:
H2 doesn't support it. DuckDB will add.

Reply:
Should support H2 as a UDF

Reply:
Thanks for the suggestion, will do